webdriver/tests/sessions/new_session/invalid_capabilities.py

```diff
@@ -83,6 +88,11 @@ def test_invalid_values(new_session, body, key, value):
                                   lambda key, value: {"firstMatch": [{key: value}]}])
 @pytest.mark.parametrize("key", invalid_extensions)
 def test_invalid_extensions(new_session, body, key):
```

Comment:
You need to change the signature here, right?

Reply:
Right